vm/vm_jit.go

```diff
@@ -1,31 +1,270 @@
+// +build evmjit
+
 package vm
 
 import "math/big"
+import "github.com/ethereum/go-ethereum/crypto"
+import "github.com/ethereum/go-ethereum/state"
+
+/*
+#include "../evmjit/libevmjit/interface.h"
+
+#cgo LDFLAGS: -L/home/chfast/go/src/github.com/ethereum/go-ethereum/evmjit/build/libevmjit -levmjit
```

Comment:
For sake of review (and development) I think it's acceptable to have it as it is. We should, as @fjl suggested, depend on the library being installed in `/usr/lib` for example.

The go-evm-llvm bridge however should reside with in the `go-ethereum` repository. Perhaps we can even have a `pkg-config` for evm-llvm and have a fallback to the current `debug_vm` if a user doesn't have evm(-llvm) installed.